content/22.vaccines.md

```diff
@@ -160,58 +160,111 @@ Modern technologies such as mRNA vaccines and vectored vaccines seem to have bee
-
-CoronaVac has been approved for use in China and has been granted emergency use in Azerbaijan, Brazil, Cambodia, Chile, Colombia, Ecuador, Hong Kong, Indonesia, Laos, Malaysia, Mexico, Philippines, Thailand, Turkey, Ukraine, and Uruguay [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinovac].<!--To Do: clean citation-->
-Sinovac has reported that their platform now has the capacity to provide up to a billion doses [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinovac].<!--To Do: clean citation-->
-<!--To Do: A sentence or two comparing to Sinopharm (2 candidates), https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7561304/ BBIBP-CorV; https://pubmed.ncbi.nlm.nih.gov/32778225/ --->
-The Sinopharm-Beijing Institute vaccine is currently approved for use in Bahrain, China, and the United Arab Emirates, but has been granted emergency use in Argentina, Cambodia, Egypt, Guyana, Hungary, Iran, Iraq, Jordan, Nepal, Pakistan, Peru, Venezuela, and Zimbabwe, with limited use in both Serbia and the Seychelles [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#sinopharm].<!--To do: clean citation-->
-In contrast, the Sinopharm-Wuhan vaccine, which has been approved for use in China since February 25th, 2021, has been distributed almost exclusively within China, with limited supplies distributed to the United Arab Emirates [@url:https://www.nytimes.com/interactive/2020/science/coronavirus-vaccine-tracker.html#wuhan].<!--To Do: clean citation-->
-<!--To Do: This should be slower to produce, I think? How does this compare to other platforms?-->
-Delays in vaccine distribution have also caused issues, particularly in Turkey where 10 million doses of Sinovac were due to arrive by December 2020, but instead only 3 million were delivered in early January [@url:https://www.nytimes.com/2021/01/25/business/china-covid-19-vaccine-backlash.html].
-Similar delays and shortages of doses promised have been reported by officials in the Philippines, Egypt, Morocco, and the United Arab Emirates [@url:https://abcnews.go.com/Health/wireStory/philippines-receive-covid-19-vaccine-delays-76163594; @url:https://www.wsj.com/articles/chinas-covid-19-vaccine-makers-struggle-to-meet-demand-11612958560].
-This will be concerning to China who have vaccine contracts for millions of doses with Indonesia (>100 million), Brazil (100 million), Chile (60 million), Turkey (50 million), Egypt (40 million) and many others [@url:https://www.wsj.com/articles/chinas-covid-19-vaccine-makers-struggle-to-meet-demand-11612958560].
+Safety analysis of the vaccine during the phase II trial revealed that most adverse reactions were either mild (grade 1) or moderate (grade 2) in severity. The most common symptom was pain at the injection site (9%) and fever (3%), and only 2% (n=7) of participants reported severe adverse events, though these were determined to be unrelated to the vaccine. 
```

Comment:
Will catch this tomorrow

Reply:
So actually those data came from this reference @doi:10/fx8z - but I didn't specifiy that it was the elderly cohort. So either we generalize this to the entire population and look at the lancet paper for the % fever instead, or report both general population and elderly symptoms. What do you think?

Reply:
I added a sentence above -- does that work? My main goal was to dodge having to get into all the subtle distinctions among groups, since overall the patterns were so similar!